test/unit/vl-map-layer.test.html

```diff
@@ -219,6 +219,15 @@
         assert.equal(layer.layer.getSource().getFeatures().length, 2);
       });
 
+      test('kan programmatorisch features toevoegen', async () => {
+        const map = fixture('vl-map-fixture');
+        const layer = map.querySelector('vl-map-layer');
+        await map.ready;
+        assert.equal(layer.layer.getSource().getFeatures().length, 1);
+        layer.addFeatureCollection('{"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"Point","coordinates":[148055,197908]},"properties":null,"id":2}]}');
```

Comment:
kunt evt. ook wel verwachten van de client dat ze het al als JSON geparsed hebben, zodat het een object is wat ze doorgeven?

Reply:
Besproken met @coemans en hij wou liever geojson in de API. Evt te bekijken bij het getten van features dan maar voorlopig doen we setten via geojson.